gcloud-java-bigquery/src/main/java/com/google/gcloud/spi/DefaultBigQueryRpc.java

```diff
@@ -64,6 +76,9 @@
   public static final String DEFAULT_PROJECTION = "full";
   // see: https://cloud.google.com/bigquery/troubleshooting-errors
   private static final Set<Integer> RETRYABLE_CODES = ImmutableSet.of(500, 502, 503, 504);
+  private static final String BASE_RESUMABLE_URI =
+      "https://www.googleapis.com/upload/bigquery/v2/projects/";
+  private static final int HTTP_RESUME_INCOMPLETE = 308;
```

Comment:
Do you mind adding a link for this error code as a comment?  It doesn't see to be on the same table as the others, and it'd be good to have it documented.